src/mpc/inputs.jl

```diff
@@ -69,7 +73,34 @@ function MPCInputs(s::MPCScenario)
     cop = Dict("ExistingChiller" => s.cooling_load.cop)
     thermal_cop = Dict{String, Float64}()
     ghp_options = 1:0
+
+    # Set up heating loads
     heating_loads = Vector{String}()
+    heating_loads_kw = Dict{String, Array{Real,1}}()
+    if !isnothing(s.process_heat_load)
```

Comment:
Why are DHW and space heating loads not created here as well, like in the non MPC case (https://github.com/NREL/REopt.jl/blob/h2-tldrd-mpc/src/core/reopt_inputs.jl#L201)?